src/__tests__/transformers.test.ts

```diff
@@ -160,6 +178,24 @@ describe('allow_properties', () => {
     expect(simpleEvent.nest1.nest2.nest3.nest4).toStrictEqual({ nest5: 'hai :3' })
   })
 
+  test('allow_properties should filter within arrays', () => {
```

Comment:
Should we add a test for dropping multiple (just in case a regression)

    simpleEvent.properties = {
      product: [
        { id: 2523532, category: 'Clothing', foo: 123 },
        { id: 875135, category: 'Sports', foo: 556 }
      ]
    }
    transformer.config = { drop: { 'properties.product': ['id', 'foo'] } }

Reply:
We can, but I don't feel it's necessarily warranted (there was not an existing test for a similar case either)